lib/main.dart

```diff
@@ -99,14 +130,15 @@ class _QuizAreaState extends State<QuizArea> {
                     answerButton(answer: questionBank.getQuestionAnswer()[0]),
```

Comment:
that could be simplified to loop

Reply:
yes and no, i mean 4 elements  don't look so bad, or do they?